lisa/schema.py

```diff
@@ -369,8 +369,24 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         default=search_space.IntRange(min=512),
         metadata=metadata(decoder=search_space.decode_count_space),
     )
-    disk_count: search_space.CountSpace = field(
-        default=search_space.IntRange(min=1),
+    data_disk_count: search_space.CountSpace = field(
+        default=search_space.IntRange(min=0),
+        metadata=metadata(decoder=search_space.decode_count_space),
+    )
+    data_disk_caching_type: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=metadata(
+            validate=validate.OneOf(
+                [
+                    constants.DATADISK_CACHING_TYPE_NONE,
+                    constants.DATADISK_CACHING_TYPE_READONLY,
+                    constants.DATADISK_CACHING_TYPE_READYWRITE,
+                ]
+            ),
+        ),
+    )
+    data_disk_iops: search_space.CountSpace = field(
+        default=search_space.IntRange(min=120, max=20000),
```

Comment:
The default value can be `min=1`. The max is not necessary. We just set a value work, but not limit it in the default value.